modules/rosapps/applications/sysutils/fontsub/lang/zh-HK.rc

```diff
@@ -0,0 +1,104 @@
+/*
+ * PROJECT:     ReactOS Font Substitute Editor
+ * LICENSE:     CC0-1.0 (https://spdx.org/licenses/CC0-1.0)
+ * PURPOSE:     Chinese (Hong Kong) resource file
+ * TRANSLATORS: Copyright 2021-2022 Chan Chilung <eason066@gmail.com>
+ * REFERENCES:  Chinese (Traditional) resource file
+ */
+
+/* UTF-8 */
+#pragma code_page(65001)
```

Comment:
This is kind of redundant, since there is the main pragma in the fontsub_res.rc file already.